test/functional/feature_assumeutxo.py

```diff
@@ -307,6 +306,8 @@ def run_test(self):
         self.test_snapshot_block_invalidated(dump_output['path'])
 
         self.log.info(f"Loading snapshot into second node from {dump_output['path']}")
+        # This node's tip is on an ancestor block of the snapshot, which should
+        # the normal case
```

Comment:
**Nit:** Missing a "be", i.e. "which should *be* the normal case." Not sure if it's worth fixing, but just wanted to point it out

Reply:
fixed, thanks!